modules/SofaGeneralLoader/src/SofaGeneralLoader/initSofaGeneralLoader.cpp

```diff
@@ -33,6 +34,11 @@ void initSofaGeneralLoader()
     static bool first = true;
     if (first)
     {
+        // msg_deprecated("SofaGeneralLoader") << "SofaGeneralLoader is deprecated; please load Sofa.Component.IO.Mesh and/or Sofa.Component.IO.Misc instead.";
```

Comment:
```suggestion
        // msg_deprecated("SofaGeneralLoader") << "SofaGeneralLoader is deprecated. It will be removed at v23.06. Use Sofa.Component.IO.Mesh and/or Sofa.Component.IO.Misc instead.";
```